packages/next-sdk/pageToolBridge.ts

```
@@ -0,0 +1,264 @@
+        if (_navigator) {
+          await _navigator(route)
+        }
+        const readyHandler = (event: MessageEvent) => {
+          if (event.source === window && event.data?.type === MSG_PAGE_READY && event.data.route === route) {
+            window.removeEventListener('message', readyHandler)
+            sendCall()
+          }
+        }
+        window.addEventListener('message', readyHandler)
+      }
+    })
```

Biome (2.4.4)

[error] Promise executor functions should not be `async`.

(lint/suspicious/noAsyncPromiseExecutor)